students/kbindhu/session06/weapon.py

```diff
@@ -0,0 +1,37 @@
+def score(weapon_type,weaponsize):
+    score_val=weapon_type(weaponsize)
+    return score_val
+
```

Comment:
Do you need the intermediate value `score_val` here?  Think about how you might collapse lines 2 and 3 into a single line.